pbm/topo/cluster.go

```diff
@@ -228,3 +229,27 @@ func GetBalancerStatus(ctx context.Context, m connect.Client) (*BalancerStatus,
 	}
 	return inf, nil
 }
+
+func ListShardedTimeseries(ctx context.Context, conn connect.Client) ([]string, error) {
+	cur, err := conn.MongoClient().
+		Database("config").Collection("collections").
+		Find(ctx,
+			bson.D{{"timeseriesFields", bson.M{"$exists": 1}}},
+			options.Find().SetProjection(bson.D{{"_id", 1}}))
+	if err != nil {
+		return nil, errors.Wrap(err, "find")
+	}
+	defer cur.Close(context.Background())
```

Comment:
```suggestion
	defer cur.Close(ctx)
```
Existing context should be propagated here.